logos/src/logos/model_string_parser.py

```
@@ -0,0 +1,86 @@
+                for kv in vals.split("_"):
+                    k, v = kv.split("=", maxsplit=1)
+                    if k not in POLICY_FIELDS:
+                        raise ValueError("Misplaced attribute in policy fields: %s. Allowed attributes: %s", k, POLICY_FIELDS)
+                    if k == "privacy" and v not in PRIVACY_VALUES:
+                        raise ValueError("Value '%s' not allowed as privacy value. Allowed types: %s", v, PRIVACY_VALUES)
+                    policy[k] = v
+            else:
+                k, v = param.split("=", maxsplit=1)
+                extra[k] = v
+    except ValueError as e:
+        raise SyntaxError("Invalid logos model string") from e
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)